src/skvhttpclient/src/skvhttp/mpack/MPackSerialization.h

```diff
@@ -573,8 +623,8 @@ class MPackWriter {
         return true;
     }
 private:
-    char* _data;
-    size_t _size;
-    mpack_writer_t _writer;
+    char* _data = NULL;
```

Comment:
use {0} syntax instead of = NULL;
same for _size

Reply:
Done